.gitignore

```diff
@@ -36,3 +36,7 @@ plugins/uefi-dbx/tests/
 .buildconfig
 .ossfuzz
 *.rej
+
+node_modules
+package-lock.json
+package.json
```

Comment:
Please end with a blank new line

Reply:
Sure?

Reply:
It previously did before your change, it should continue to.